### ISSUE TITLE
train.csv and valid.csv could not be found in the repo

### ISSUE THREAD

**Issue by kidus-admassu** [WeixiongLin/PMC-CLIP#11]
Dear Authors,

I wanted to train the model on the PMC-OA dataset, but I couldn't locate `train.csv` and `valid.csv` in the codebase. There was no explanation on this in the documentation so, could you please elaborate more on this?

Thanks,

Kidus

**Comment by WeixiongLin** [comment]
Hi @kidus-admassu , the `train.csv` and `valid.csv` is to illustrate the training and validation set. We refer you to [PMC-OA Dataset](https://huggingface.co/datasets/axiong/pmc_oa) which is released for PMC-CLIP pretraining.

**Comment by WeixiongLin** [comment]
Notice that PMC-OA is a rather large dataset, if you just want to validate the code, you could use [ROCO](https://www.kaggle.com/datasets/virajbagal/roco-dataset) as input data.

**Comment by PixelChen24** [comment]
> Notice that PMC-OA is a rather large dataset, if you just want to validate the code, you could use [ROCO](https://www.kaggle.com/datasets/virajbagal/roco-dataset) as input data.

In the hugging face link provided by you, I cannot find any file that ends with csv
